mirrulations-core/src/mirrcore/data_storage.py

```diff
@@ -8,6 +8,7 @@ def __init__(self):
         self.documents = database['documents']
         self.comments = database['comments']
         self.attachments = database['attachments']
+        self.extraction = database['extracted_text']
```

Comment:
To align with our disk storage we may want the database to be named `comments_extracted_text`

Reply:
Since we may eventually be adding tables for `documents_extracted_text`